### PR TITLE
Add finalize command

### PR DESCRIPTION
**Pull request by djones**
This command allows you to run a parallelized build and run a finalize all step at the very end of your test run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ USAGE
 # Commands
 <!-- commands -->
 * [`percy-agent exec`](#percy-agent-exec)
```
Comment (djones) [inline README.md]: Note that changed in this readme are all autogenerated.

File: src/commands/finalize.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  ]
+
+  buildService: BuildService = new BuildService()
+
+  async run() {
+    const {flags} = this.parse(Finalize)
+
+    if (this.percyEnvVarsMissing()) { return }
+
+    if (!process.env.PERCY_PARALLEL_NONCE) {
```
Comment (djones) [inline src/commands/finalize.ts]: I think at some point we could make this generic and have a function that takes in all the required env vars, outputs to the log if they're missing and exits. But for now this'll do.

File: README.md (hunk 1 above)
Comment (fotinakis) [reply README.md]: waat

**Comment by djones** [comment]
Exit code 1 is missing env var.

```bash
$ percy-agent finalize --all
 ›   Error: You must set PERCY_PARALLEL_NONCE
$ echo $?
1
```

Exit code 2 is flag is missing. (2 is what Oclif have selected for this case)

```bash
$ percy-agent finalize
 ›   Error: Missing required flag:
 ›     -a, --all
 ›   See more help with --help
$ echo $?
2
```

Successful build now outputs the URL:

```bash
$ PERCY_PARALLEL_NONCE=madethisup percy-agent finalize -a
[percy] Finalized parallel build.
[percy] Visual diffs are now processing: https://percy.io/test/test/builds/1064826
```

409's no longer say "[percy] Finalized parallel build.":

```bash
$ PERCY_PARALLEL_NONCE=madethisup percy-agent finalize -a
[percy] StatusCodeError 409 - {"errors":[{"status":"conflict","detail":"Can only finalize pending builds"}]}
```

**Pull request merged**